heartbeat/data/heartbeatdata.go

```diff
@@ -11,12 +11,9 @@ import (
 type PubKeyHeartbeat struct {
 	PublicKey       string    `json:"publicKey"`
 	TimeStamp       time.Time `json:"timeStamp"`
-	MaxInactiveTime Duration  `json:"maxInactiveTime"`
```

Comment:
the entries still exist on `heartbeat/data/heartbeat.pb.go` . is it safe and compatible to remove the fields from there as well?

Reply:
no, let's leave them as they are until the hb refactor is complete